Src/Base/AMReX_Utility.cpp

```diff
@@ -116,7 +116,13 @@ bool
 amrex::UtilCreateDirectory (const std::string& path,
                             mode_t mode, bool verbose)
 {
-    return FileSystem::CreateDirectories(path, mode, verbose);
+    double sleep = 0.1;
+    while (sleep < 2.0) {
+        if (FileSystem::CreateDirectories(path, mode, verbose)) { return true; }
```

Comment:
Maybe if this fails, it could do, e.g.:
```
amrex::Print() << "FileSystem::CreateDirectories() failed with errno: " << std::strerror(errno) << ". Retrying.\n";
```

Reply:
If verbose is true, `FileSystem::CreateDirectories` will print message. In most cases, we are supposed to have a way to not print.

Reply:
We are trying to follow this policy. https://github.com/xsdk-project/xsdk-community-policies/blob/master/package_policies/M11.md